app/services/control_rate_service.rb

```diff
@@ -75,7 +76,8 @@ def registration_counts
     return @registration_counts if defined? @registration_counts
 
     @registration_counts =
-      region.assigned_patients
+      region.registered_patients
+        .with_hypertension
```

Comment:
....and we're back to the good ole' days where registration counts are really registrations! 😁